deploy-board/deploy_board/templates/clusters/base_images_events.html

```diff
@@ -32,11 +32,15 @@ <h4 class="panel-title pull-left">Base Images Events</h4>
     </button>
     {% endif %}
     </div>
+    <div class="row">
+    <button class="deployToolTip btn btn-default btn-block" id="cancelBtnId">   
+    <span class="glyphicon glyphicon-remove-circle"></span> Cancel
+    </button>
+    </div>
```

Comment:
Needs some formatting.

Reply:
Also, I think we should
1. only show this when demote is present. 
2. or better, when demote is present and there is any event is in the INIT state. So if every event is COMPLETED, can't cancel.

Reply:
yes, formatting makes sense.
and in my opinion, we only need to check the events. if there is any event with INIT state, we have cancel button.